src/main/kotlin/no/nav/klage/oppgave/clients/joark/JoarkClient.kt

```diff
@@ -0,0 +1,166 @@
+package no.nav.klage.oppgave.clients.joark
+
+import brave.Tracer
+import no.nav.klage.oppgave.clients.ereg.EregClient
+import no.nav.klage.oppgave.clients.pdl.PdlFacade
+import no.nav.klage.oppgave.domain.joark.*
+import no.nav.klage.oppgave.domain.klage.Klagebehandling
+import no.nav.klage.oppgave.domain.klage.PartIdType
+import no.nav.klage.oppgave.service.TokenService
+import no.nav.klage.oppgave.util.getLogger
+import no.nav.klage.oppgave.util.getSecureLogger
+import org.apache.http.HttpHeaders
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.multipart.MultipartFile
+import org.springframework.web.reactive.function.client.WebClient
+import java.util.*
+
+@Component
+class JoarkClient(
+    private val joarkWebClient: WebClient,
+    private val tokenService: TokenService,
+    private val tracer: Tracer,
+    private val pdlFacade: PdlFacade,
+    private val eregClient: EregClient
+) {
+    companion object {
+        @Suppress("JAVA_CLASS_ON_COMPANION")
+        private val logger = getLogger(javaClass.enclosingClass)
+        private val secureLogger = getSecureLogger()
+        private const val BREV_TITTEL = "Brev fra Klageinstans"
+        private const val BREVKODE = "BREV_FRA_KLAGEINSTANS"
+        private const val BEHANDLINGSTEMA_KLAGE_KLAGEINSTANS = "ab0164"
```

Comment:
Enhet kan ikke hardkodes. Den hentes fra Axsys, men så kan man jo være knyttet til mer enn en enhet. Det kan hende flere av de verdiene du hardkoder her bør komme i `VedtakFullfoerInput`. Kanskje bare legge opp til det med en gang?

Reply:
Enig. Gjelder det samme også for behandlingstemaet i linja over kanskje?

Reply:
Skal endre disse, de henger igjen fra ubeskyttede direktekall som jeg har brukt tidligere i utviklinga.

Reply:
Ser gjennom nå, og ser at det vel egentlig bare er enhet, og muligens BREV_TITTEL, som gir mening å kunne endre for saksbehandler. Skal lage en løsning for enhet.f